public/index.html

```diff
@@ -5,79 +5,108 @@
-    <link rel="stylesheet" href="style.css">
 
     <title>Project Fluent</title>
   </head>
   <body>
-    <div class="main">
-      <header class="header">
-        <h1 class="header__wordmark">
-          fluent
+    <div>
+      <nav id="topbar">
+        <div class="logo">
+          <a href="/">fluent</a>
+        </div>
+        <ul>
+          <li>
+            <a href="#">Home</a>
+          </li>
+          <li>
+            <a href="/fluent/guide">Documentation</a>
+          </li>
+          <li>
+            <a href="/play">Online Editor</a>
+          </li>
+          <li>
+            <a href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <div>
+      <header id="intro">
+        <h1>
+          Fluent
         </h1>
-        <nav class="header__nav">
-          <a class="link" href="http://projectfluent.io/fluent/guide">Syntax Guide</a>
-          <a class="link" href="http://projectfluent.io/play">Online Editor</a>
-          <a class="link" href="https://github.com/projectfluent">GitHub</a>
-          <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Mailing list</a>
-          <a class="link" href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
-        </nav>
+        <h2>A localization paradigm designed to unleash the expressive power of the natural language.</h2>
+        <a href="https://github.com/projectfluent/fluent/wiki/Integrating-Fluent" class="cta">Get Started</a>
       </header>
-
-      <section class="section section--centered">
-        <p class="pitch">
-
-          <strong class="highlight">Fluent</strong> is a localization paradigm
-          designed to unleash the expressive power of the natural language.
-
+    </div>
+    <div>
+      <main>
+        <ul>
+          <li>
+            <h1>
+              Innovative
+            </h1>
+            <p>Natural-sounding translations that use the grammatical features of each language..</p>
+          </li>
+          <li>
+            <h1>
+              Powerfull & Extensible
+            </h1>
+            <p>
+              Date, time, number, plural rules support. Full bidirectionality.
+              Any internationalization formatters can be added.
+            </p>
+          </li>
+          <li>
+            <h1>
+              Flexible
+            </h1>
+            <p>
+              Can translate on client-side and server-side. Live language switching,
+              robust error handling.
+              JS, Python, Rust and React implementation.
+            </p>
+          </li>
+          <li>
+            <h1>
+              Complete
+            </h1>
+            <p>
+              Visual localization editor, linters, checkers and
+              workflow tooling.
+            </p>
+          </li>
+        </ul>
+        <aside id="demo"></aside>
+      </main>
+    </div>
+    <div>
+      <section id="explainer">
```

Comment:
I tried changing the style of this block in devtools and it's lighter and cleaner without the dark background. Can you try it too and see if you like it better?

Reply:
Like this: https://screenshots.firefox.com/RvHU4APjkHQ2Oetl/localhost ?

No, not really. And I like the contrast when the examples are added, so would prefer to keep it.

Reply:
Yeah, like this. This avoids the situation in which the main content is trapped between two dark ribbons. Not blocking on this.

Reply:
I'd like to evaluate it once we re-add the examples. I was planning to ask Matjaz to take a look and apply any minor/color balancing tune ups then.